lib/file-info-view.coffee

```diff
@@ -10,10 +12,29 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?() or 'untitled')
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
```

Comment:
I think the FileInfoView doesn't have to be about showing only the path itself. What if someone adds there file permissions or owner/group information or something like that? That is still considered an information about the file, but it is not path related.

Also currently it contains the modifier flag (`*`) currently which is not behaving as a link (not underlinked upon hovering over it) and yet it is clickable.

I know I am probably nitpicking, but now after I closed my pull request I have to focus my energy somewhere, sorry for that :blush: .